configs/platforms/ubuntu-25.04-aarch64.rb

```diff
@@ -0,0 +1,14 @@
+platform "ubuntu-25.04-aarch64" do |plat|
+  plat.inherit_from_default
+
+  packages = %w(
+    libbz2-dev
+    libreadline-dev
+    libselinux1-dev
+    gcc
+    swig
+    systemtap-sdt-dev
+    zlib1g-dev
+  )
+  plat.provision_with "export DEBIAN_FRONTEND=noninteractive && apt-get update -qq && apt-get install -qy --no-install-recommends #{packages.join(' ')}"
```

Comment:
Because this is identical to what is in the platform default in vanagon, I think we can remove this. If you see that in the other Ubuntu flavors, mind removing those as well?